build/build.xml

```diff
@@ -85,7 +85,7 @@
         </javac>
 
         <mkdir dir="../android/temp/zip"/>
-        <unzip src="../android/libs/android-support-v4.jar" dest="../android/temp/zip"/>
+        <!-- <unzip src="../android/libs/android-support-v4.jar" dest="../android/temp/zip"/> -->
```

Comment:
reverting this. irrelevant to the work at hand and could possibly cause trouble.